### PR TITLE
fix(virtualRepeat): Virtual repeat starting off empty #3807

### PR DESCRIPTION
**Pull request by kseamon**

### CODE REVIEW

**Comment by kseamon** [comment]
@jelbourn

File: src/components/virtualRepeat/virtualRepeater.js [hunk 1]
```diff
@@ -366,6 +366,12 @@ function VirtualRepeatController($scope, $element, $attrs, $browser, $document,
   this.itemsLength = 0;
 
   /**
+   * @type {!Function} Unwatch callback for item size (when md-items-size is
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtualRepeater.js]: Could be `function(): void`

File: src/components/virtualRepeat/virtualRepeater.js [hunk 2]
```diff
@@ -438,6 +456,7 @@ VirtualRepeatController.prototype.containerUpdated = function() {
   }
 
   if (!this.sized) {
+    this.unwatchItemSize_();
```
Comment (jelbourn) [inline src/components/virtualRepeat/virtualRepeater.js]: Do you think there should be a test for the watcher being deregistered?

**Comment by jelbourn** [comment]
Overall LGTM

**Pull request closed without merging**